SparkPost/2web2ui
# AC-851: Pending Cancellation Banner

### What Changed
 - Added pending cancellation banner

### How To Test
 - Go to `/account/settings` and cancel account
 - Check that account cancellation banner shows up on `/account/settings` or `/account/billing`
 - Check that banner close hides banner
 - Check that banner shows up within 7 days (will figure out best way to test this)

### To Do
